packages/slate-react/src/plugins/after.js

```diff
@@ -553,24 +553,27 @@ function AfterPlugin() {
     // COMPAT: If the selection is at the end of a non-void inline node, and
     // there is a node after it, put it in the node after instead. This
     // standardizes the behavior, since it's indistinguishable to the user.
-    if (
-      anchorInline &&
-      !anchorInline.isVoid &&
-      anchorOffset == anchorText.text.length
-    ) {
-      const block = document.getClosestBlock(anchorKey)
-      const next = block.getNextText(anchorKey)
-      if (next) range = range.moveAnchorTo(next.key, 0)
-    }
+    if (range.isExpanded) {
+      if (
+        range.isExpanded &&
```

Comment:
Isn't this extra `range.isExpanded` unnecessary with the outer `if`?

Reply:
My mistake.  I will delete that.